geoutils/georaster.py

```diff
@@ -280,9 +280,14 @@ def data(self, new_data):
             raise ValueError("New data must be a numpy array.")
 
         # Check that new_data has correct shape
-        if new_data.shape != self._data.shape:
+        if self.is_loaded:
+            orig_shape = self._data.shape
+        else:
+            orig_shape = (self.count, self.height, self.width)
+
+        if new_data.shape != orig_shape:
             raise ValueError("New data must be of the same shape as\
- existing data: {}.".format(self.shape))
+ existing data: {}.".format(orig_shape))
```

Comment:
There might be something with GitHub's visualisation, but it seems to me like this line is unindented. Is that true?

Reply:
It's correct, but if I indent, the text that is printed has a large space. How do I avoid this?

Reply:
I think I found it. Should I just add the quotes at the end and beginning of each newline?

Reply:
See commit 63e5717